### ISSUE TITLE
root-config template - isLocal in not check correctly in webpack.config.js

### ISSUE THREAD

**Issue by hc-ribeiro** [single-spa/create-single-spa#252]
Hi, I believe the file https://github.com/single-spa/create-single-spa/blob/main/packages/generator-single-spa/src/root-config/templates/webpack.config.js have an error in line 22, because the ```webpackConfigEnv.isLocal``` is tested as string, causing error when run the project on local envoriment.

```
resolve.js:12 Uncaught (in promise) Error: Unable to resolve bare specifier '@bl/root-config' from http://localhost:9000/ (SystemJS Error#8 https://git.io/JvFET#8)
    at resolve.js:12
    at c.O.resolve (resolve.js:8)
    at system-core.js:34
```

When remove the double quotes of ```"true"```, as below, the project runs fine.

 ``` isLocal: webpackConfigEnv && webpackConfigEnv.isLocal ===true```

**Comment by joeldenning** [comment]
Hi @hc-ribeiro, thanks for the bug report. This is a duplicate of https://github.com/single-spa/create-single-spa/issues/250, which will be fixed in #251. The workaround you shared above is the change that's in #251.

**Comment by joeldenning** [comment]
Fix released in https://github.com/single-spa/create-single-spa/releases/tag/v2.0.2
